app/views/rubric_criteria/_rubric_criterion_level.html.erb

```diff
@@ -4,13 +4,20 @@
 <div class="rubric_level_header">
   <%= f.label "level_#{rubric_level_index}_name".to_sym,
               I18n.t("rubric_criteria.level.level_index",
-		     index: rubric_level_index),
-	      class: "bold_inline_label" %>
+                     index: rubric_level_index),
+              class: "bold_inline_label" %>
 </div>
 <br>
-  <%= f.text_field "level_#{rubric_level_index}_name".to_sym,
-		   class: "rubric_level_input"%>
-  <%= f.text_area "level_#{rubric_level_index}_description".to_sym,
-		  class: "rubric_text_area"%>
+<span class="rubric_level_title">Name:  </span>
```

Comment:
Don't use hard-coded strings; instead, use internationalized versions. Here you should look up how to add new strings for `Level` attributes, which you can do [here](https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models). (Note that we store the locale files for models under `config/locales/models`.